recipe/meta.yaml

```
@@ -10,19 +10,21 @@ source:
   sha256: 0dbd14a8aef6636764f88a8fd1fcc9a91921e5c50356e6aab251782f264ae960
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
   number: 0
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
     - hatchling >=1.0
+    - wheel
   run:
-    - python >=3.7
+    - python
     - hatchling >=1.0
```

```suggestion
    - hatchling >=1.5
```
https://github.com/jupyterlab/hatch-jupyter-builder/blob/v0.8.3/pyproject.toml#L27